Userland/Libraries/LibJS/Runtime/Temporal/Duration.cpp

```diff
@@ -975,7 +975,7 @@ ThrowCompletionOr<DateDurationRecord> balance_duration_relative(VM& vm, double y
     return create_date_duration_record(years, months, weeks, days);
 }
 
-// 7.5.21 AddDuration ( y1, mon1, w1, d1, h1, min1, s1, ms1, mus1, ns1, y2, mon2, w2, d2, h2, min2, s2, ms2, mus2, ns2, relativeTo ), https://tc39.es/proposal-temporal/#sec-temporal-addduration
+// 7.5.22 AddDuration ( y1, mon1, w1, d1, h1, min1, s1, ms1, mus1, ns1, y2, mon2, w2, d2, h2, min2, s2, ms2, mus2, ns2, relativeTo ), https://tc39.es/proposal-temporal/#sec-temporal-addduration
```

Comment:
Don't change this here, change this where the preceding AO is actually being removed

Reply:
Ah, alright. Will keep this in mind for the next time.